_locales/cs/joystickbit-strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "joystickbit.ButtonType.down|block": "stisknutí",
+  "joystickbit.ButtonType.up|block": "uvolnění",
+  "joystickbit.JoystickBitPin.P12|block": "C",
+  "joystickbit.JoystickBitPin.P13|block": "D",
+  "joystickbit.JoystickBitPin.P14|block": "E",
+  "joystickbit.JoystickBitPin.P15|block": "F",
+  "joystickbit.Vibration_Motor|block": "vibruj po dobu %time ms",
+  "joystickbit.getButton|block": "je tlačítko %button stisknuto",
+  "joystickbit.getRockerValue|block": "hodnota osy %rocker u levé páčky",
+  "joystickbit.initJoystickBit|block": "inicializuj joystick:bit",
+  "joystickbit.onButtonEvent|block": "při %event tlačítka %button",
+  "joystickbit.rockerType.X|block": "X",
+  "joystickbit.rockerType.Y|block": "Y",
+  "joystickbit|block": "Joystickbit",
+  "{id:category}Joystickbit": "Joystickbit"
```

Comment:
This is just a nitpick, but I'd personally use the original name - "Joystick:bit" just to be consistent with the "inicializuj joystick:bit" and "Bezdrátově ovládej své výtvory pomocí joystick:bit."

Reply:
I thought so too at first. But on second thought, all the other Elecfreaks extensions do not add the `colon` to the name 😞 

<img width="198" alt="Screenshot 2024-11-11 at 5 53 00" src="https://github.com/user-attachments/assets/d94348ef-a9e3-493a-9c24-e80ce6cfc7b7">